components/EditTagsModal.tsx

```diff
@@ -0,0 +1,164 @@
+    } catch (e) {
+      addToast({
+        type: TOAST_TYPE.ERROR,
+        message: (
+          <FormattedMessage
+            defaultMessage="Error submiting form: {error}"
+            values={{
+              error: e.message,
+            }}
+          />
+        ),
+      });
+      return;
+    }
+    addToast({
+      type: TOAST_TYPE.SUCCESS,
+      message: <FormattedMessage defaultMessage="Successfully updated tags" />,
+    });
+    handleClose();
+  };
+
+  const handleClose = () => {
+    onClose?.();
+  };
+
+  return (
+    <StyledModal flexGrow={1} maxWidth="500px" onClose={handleClose} trapFocus>
+      <Formik initialValues={initialValues} onSubmit={submit}>
+        {formik => (
+          <Form onSubmit={formik.handleSubmit}>
+            {/* A hack to disable automatic focus from Formik on the tags input.  */}
+            {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+            <div tabIndex={0} />
+            <ModalHeader onClose={handleClose} hideCloseIcon={true}>
+              <FormattedMessage defaultMessage="Edit tags" />
+            </ModalHeader>
+            <ModalBody mb={0}>
+              <StyledInputFormikField
+                name="tags"
+                htmlFor="tags"
+                labelProps={{ fontWeight: 500, fontSize: '14px', lineHeight: '17px' }}
+              >
+                {({ field }) => {
+                  return (
+                    <CollectiveTagsInput
+                      {...field}
+                      defaultValue={formik.values.tags}
+                      onChange={tags => {
+                        formik.setFieldValue(
+                          'tags',
+                          tags.map(t => t.value.toLowerCase()),
+                        );
+                      }}
+                      suggestedTags={tagStats?.nodes?.map(node => node.tag).filter(tag => !IGNORED_TAGS.includes(tag))}
+                    />
+                  );
+                }}
+              </StyledInputFormikField>
+              <MessageBox type="info" mt={3}>
+                <FormattedMessage
+                  id="collective.tags.info"
+                  defaultMessage="Tags help you improve your group’s discoverability and connect with similar initiatives across the world."
+                />
+              </MessageBox>
+            </ModalBody>
+            <ModalFooter isFullWidth>
+              <Flex justifyContent="flex-start" flexWrap="wrap" gap="16px">
+                <StyledButton
+                  minWidth={132}
+                  buttonStyle="primary"
+                  loading={loading}
+                  disabled={loading || !formik.dirty}
+                  type="submit"
+                >
+                  <FormattedMessage id="save" defaultMessage="Save" />
+                </StyledButton>
+                <StyledButton
+                  minWidth={132}
+                  buttonStyle="secondary"
+                  loading={loading}
```

Comment:
The widespread practice in the codebase is to mark only the save button as `loading`. I personally prefer this approach, having two loading buttons look strange to me.

```suggestion
```

![image](https://user-images.githubusercontent.com/1556356/217525501-d586054a-ad5c-42da-a26a-b9b6806017cc.png)

Reply:
Ah, yes, this was unintentional, must have missed removing this when copying the Save button.